plugins/common/wayfire/plugins/common/move-snap-helper.hpp

```diff
@@ -140,6 +140,27 @@ class move_snap_helper_t : public wf::custom_data_t
         }
 
         view->disconnect_signal("geometry-changed", &view_geometry_changed);
+        /**
+         * Restore the fullscreen state so the window is
+         * not in a weird position afterwards, fullscreen
+         * are by spec the geometry of the output (or span outputs)
+         */
+        if (view->fullscreen)
+        {
+            wf::output_t *output = wf::get_core().get_active_output();
```

Comment:
you should use `view->get_output()`

Reply:
I haven't tested it yet, but I'm not sure what's supposed to happen
If I use the views output and I drag the fullscreen view 
from output A (no expo) to output B (expo active),
I would need to include some calculation for the ws on output B?

Reply:
if you drag the view to output B then view->get_output() will already point to output B, this is why I said that the view's output shoukd be used.

Reply:
also don't worry about expo or no expo, the move-snap-helper takes care of that